doc/source/security/apparmor.rst

```diff
@@ -8,161 +8,173 @@ The operating system has a Discretionary Access Controls (DAC) system. AppArmor
+A profile is in one of the following modes:
 
 * Enforce - the default setting, applications are prevented from taking actions restricted by the profile rules.
 
 * Complain - applications are allowed to take restricted actions, and the actions are logged.
 
 * Disabled - Applications are allowed to take restricted actions, and the actions are not logged. 
 
- You can mix enforce profiles and complain profiles. 
+ You can mix enforce profiles and complain profiles in your server. 
 
-Checking the Current Status
+Install the Utilities used to control AppArmor
+------------------------------------------------
+
+Install the ``apparmor-utils`` package to work with profiles. Use these utilities to create, update, enforce, switch to complain mode, and disable profiles, as needed:
+
+    .. sourcecode:: bash
+
+        $ sudo apt-get -y install apparmor-utils
+        Reading package lists... Done
+        Building dependency tree
+        ...
+        The following additional packages will be installed:
+            python3-apparmor python3-libapparmor
+        ...
+
+
+Check the Current Status
 -----------------------------
 
 As root or using ``sudo``, you can check the AppArmor status:
 
-.. sourcecode:: bash
-
-    $ sudo aa-status
-    apparmor module is loaded.
-    34 profiles are loaded.
-    32 profiles in enforce mode.
-    ...
-        /usr/sbin/mysqld
-    ...
-    2 profiles in complain mode.
-    ...
-    3 profiles have profiles defined.
-    ...
-    0 processes are in complain mode.
-    0 processes are unconfined but have a profile defined.
+    .. sourcecode:: bash
+
+        $ sudo aa-status
+        apparmor module is loaded.
+        34 profiles are loaded.
+        32 profiles in enforce mode.
+        ...
+            /usr/sbin/mysqld
+        ...
+        2 profiles in complain mode.
+        ...
+        3 profiles have profiles defined.
+        ...
+        0 processes are in complain mode.
+        0 processes are unconfined but have a profile defined.
 
 .. _complain-one:
 
-Change a Profile to Complain mode
+Switch a Profile to Complain mode
 -----------------------------------------
 
-To improve a profile, change to the ``complain`` mode and then run any actions that generates log entries. Update the profile from the log information.
+Switch a profile to complain mode when the program is in your path with this command:
 
-To change a profile to complain mode and the program is in your path:
-
-.. sourcecode:: bash
+    .. sourcecode:: bash
 
-    $ sudo aa-complain <program>
+        $ sudo aa-complain <program>
 
-If needed, specify the path:
+If needed, specify the program's path in the command:
 
-.. sourcecode:: bash
+    .. sourcecode:: bash
 
-    $ sudo aa-complain /sbin/<program>
+        $ sudo aa-complain /sbin/<program>
 
-If the profile is not in stored in /etc/apparmor.d, use the following command:
+If the profile is not in stored in the default location, /etc/apparmor.d, use the following command:
```

Comment:
One "in" preposition seems to be excessive

Reply:
Removed - I wanted to be sure the reader could find the default location. It's mentioned in an earlier sentence.